www/app/Services/SubAgentRunner.php

```
@@ -0,0 +1,238 @@
+            'prompt' => $prompt,
+            'is_background' => $isBackground,
+        ]);
+
+        // Re-initialise the stream for this turn
+        $this->streamManager->startStream($conversation->uuid, [
+            'model' => $conversation->model,
+            'provider' => $conversation->provider_type,
+            'subagent_task_id' => $task->id,
+        ]);
+
+        ProcessConversationStream::dispatch($conversation->uuid, $prompt);
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Jobs\ProcessConversationStream::dispatch().


(staticMethod.notFound)